apps/minifront/vitest.config.ts

```diff
@@ -5,5 +5,7 @@ export default defineConfig({
     snapshotSerializers: [],
     environment: 'jsdom',
     setupFiles: ['./tests-setup.ts'],
+    silent: false,
+    reporters: 'default',
```

Comment:
Both of these are default values, can remove

Reply:
removed